test/e2e/playbook/provision.yaml

```diff
@@ -139,13 +139,23 @@
       when: ansible_facts['distribution'] == "Ubuntu"
 
     - name: Install rpm packages
-      ansible.builtin.dnf:
-        pkg:
-          - iproute-tc
-          - grubby
-        state: present
-        update_cache: true
       when: ansible_facts['distribution'] == "Fedora"
+      block:
+        - name: Common rpm packages
+          ansible.builtin.dnf:
+            pkg:
+              - iproute-tc
+              - grubby
+            state: present
+            update_cache: true
+        - name: Extra python3 rpm packages for ansible
+          ansible.builtin.dnf:
+            pkg:
+              - python3-libselinux
+              - python3-packaging
```

Comment:
I would merge this two tasks into one and create a single list of requires packages.

Reply:
The 2nd one is conditional not necessary (and neither possible) on our older Fedora distros.